src/modules/create_append.js

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line import/no-cycle
+import removeTask from './remove_task.js';
+// eslint-disable-next-line import/no-cycle
+import { saveOldContent, updateContent } from './edit_task.js';
+// eslint-disable-next-line import/no-cycle
+import { checkStatus, checkTask } from './task_status.js';
```

Comment:
Import cycles can cause performance issues, it's better not to disable the linters, instead try to refactor some of the code.